mushroomsoft/app/src/app/components/technology/technology.component.html

```diff
@@ -1,31 +1,31 @@
 <div id="tecnology" class="ms-technologies">
-    <div class="ms-container-3 ms-w-container">
-            <div class="ms-layout-grid">
-                <h1 id="tecnology" class="ms-heading-8 ms-tecnology-title">
-                {{titleTechnology}}
-                </h1>
-                <p id="ms-tecnology-description" class="ms-paragraph-18">
-                <br />{{descriptionTechnology}} <br />
-                </p>
-            </div>
-            <div *ngFor="let item of technologyItemsService" >
-                <div class="ms-layout-grid {{item.class}}">
-                    <h1 id="typeTechnology" class="ms-heading-8">
-                        {{item.title}}
-                    </h1>
-                    <div *ngFor="let items of technologyItemsImage ; let index = index">
-                        <img
-                            [id]="item.images[index].id"
-                            [name]="item.images[index].id"
-                            [src]="item.images[index].url"
-                            class="ms-technology-image"
-                            [alt]="item.images[index].id"
-                        />
-                    </div>
-                </div>
-            </div>
+  <div class="ms-container-3 ms-w-container">
+    <div class="ms-layout-grid">
+      <h1 id="tecnology" class="ms-heading-8 ms-tecnology-title">
+      {{titleTechnology}}
+      </h1>
+      <p id="ms-tecnology-description" class="ms-paragraph-18">
+      <br />{{descriptionTechnology}} <br />
```

Comment:
Delete spaces in <br` `/>, would be <br/>

Reply:
resolved

Reply:
in all occurrences, please.

Reply:
resolved